### PR TITLE
Add empty state to sbd details

### PR DESCRIPTION
**Pull request by EMaksy**
# Description
This PR will add an empty state message for the SBD Details component in the case there are no sbd devices. Instead of just rendering the title, we render a text which informs the user.

## Why? 
When using disk-based SDB fencing, the SBD/fencing section in the cluster details view shows the configured SBD devices and whether they are in healthy or unhealthy status.

When using diskless SBD fencing, there are no SBD devices. Therefore, the SBD/fencing section should inform the user with a message.

### Before:
image

### After (Check Storybook):
image

## How was this tested?
Added test and story

### CODE REVIEW

File: assets/js/pages/ClusterDetails/SBDDetails.jsx [hunk 1]
```diff
@@ -10,20 +10,24 @@ const getStatusPill = (status) =>
   );
 
 function SBDDetails({ sbdDevices }) {
+  const sbdDetailsHeader = 'SBD/Fencing';
```
Comment (arbulu89) [inline assets/js/pages/ClusterDetails/SBDDetails.jsx:13]: This consts can go outside the function, module level (or even in the html tags itself, as they are not repeated)

File: assets/js/pages/ClusterDetails/SBDDetails.test.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    expect(screen.getByText(expectedEmptyStateMsg)).toBeInTheDocument();
+  });
+
+  it('should render healthy sbd device', () => {
+    const expectedSBD = sbdDevicesFactory.buildList(1, { status: 'healthy' });
+    const [{ device: expectedDeviceName, status: expectedHealthStatus }] =
+      expectedSBD;
+    const capitalizedHealthStatus = capitalize(expectedHealthStatus);
+    render(<SBDDetails sbdDevices={expectedSBD} />);
+    expect(screen.getByText(expectedDeviceName)).toHaveTextContent(
```
Comment (arbulu89) [inline assets/js/pages/ClusterDetails/SBDDetails.test.jsx:26]: This double kind of expectation is really strange.
Here you should do `expect(screen.getByText(expectedDeviceName)).toBetTruthy()` or similar. Check that the text you got with a string contains that strings doesn't make much sense.
The same applies to other expectations

File: assets/js/pages/ClusterDetails/SBDDetails.test.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      sbdDevicesFactory.build({ status: 'unhealthy' }),
+      sbdDevicesFactory.build({ status: 'healthy' }),
+    ];
+
+    render(<SBDDetails sbdDevices={expectedSBD} />);
+
+    expectedSBD.forEach(({ device, status }) => {
+      expect(screen.getByText(device)).toBeInTheDocument();
+      expect(
+        screen.getByText(status === 'healthy' ? 'Healthy' : 'Unhealthy')
```
Comment (arbulu89) [inline assets/js/pages/ClusterDetails/SBDDetails.test.jsx:61]: Cannot you use `capitalize` here?

**Review by arbulu89 — COMMENTED** [review]
@EMaksy it looks good!
some small changes and we can merge

File: assets/js/pages/ClusterDetails/HanaClusterDetails.stories.jsx [hunk 4]
```diff
@@ -199,3 +199,14 @@ export const WithUnmanagedResources = {
     details: unmanagedNodeResources,
   },
 };
+
+export const WithNoSBDDevices = {
+  args: {
+    ...Hana.args,
+    details: {
+      ...HanaScaleOut.args.details,
```
Comment (arbulu89) [inline assets/js/pages/ClusterDetails/HanaClusterDetails.stories.jsx:207]: `Hana` here as well

File: assets/js/pages/ClusterDetails/SBDDetails.test.jsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+describe('SBDDetails', () => {
+  it('should render empty state message when sbd devices are empty', () => {
+    const expectedEmptyStateMsg = 'No additional fencing details to display.';
+    const sbdDevices = [];
+
+    render(<SBDDetails sbdDevices={sbdDevices} />);
+    expect(screen.getByText(expectedEmptyStateMsg)).toBeInTheDocument();
+  });
+
+  it.each([['healthy'], ['unhealthy']])(
```
Comment (arbulu89) [inline assets/js/pages/ClusterDetails/SBDDetails.test.jsx:20]: is this list of lists intentional? `it.each(['healthy', 'unhealthy'])(...` looks more normal.
I even think the test works by accident, as you pass a list to `status` in the factory

File: assets/js/pages/ClusterDetails/SBDDetails.test.jsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      const [{ device: expectedDeviceName, status: expectedHealthStatus }] =
+        expectedSBD;
+      const capitalizedHealthStatus = capitalize(expectedHealthStatus);
+
+      render(<SBDDetails sbdDevices={expectedSBD} />);
+
+      expect(screen.getByText(expectedDeviceName)).toHaveTextContent(
+        expectedDeviceName
+      );
+      expect(screen.getByText(capitalizedHealthStatus)).toHaveTextContent(
```
Comment (arbulu89) [inline assets/js/pages/ClusterDetails/SBDDetails.test.jsx:35]: Change this to `expect(screen.getByText(capitalize(expectedHealthStatus))).toBeTruthy()`.
The same concept above

**Review by arbulu89 — APPROVED** [review]
Thank you @EMaksy 
Some last things, but you can merge without other review from me

File: assets/js/pages/ClusterDetails/SBDDetails.test.jsx (hunk 3 above)
Comment (EMaksy) [reply assets/js/pages/ClusterDetails/SBDDetails.test.jsx]: Yes, we can!

**Review by rtorrero — APPROVED** [review]
LGTM

**Pull request merged**